plugins/wordcount/plugin.js

```diff
@@ -309,22 +333,6 @@ CKEDITOR.plugins.add("wordcount", {
             counterElement(editorInstance).className = "cke_path_item";
         }
 
-        function debounce(func, wait, immediate) {
```

Comment:
why is this removed? we need this? this is to improve the latency on the story editor? cc: @RyanCopley

Reply:
Good Catch Raghu. I might have missed to pull down the latest.